lib/image-stream/index.js

```diff
@@ -76,14 +76,13 @@ exports.getFromFilePath = (file) => {
     }
 
     return utils.getArchiveMimeType(file).then((type) => {
+      const MIME_TYPE_RAW_IMAGE = 'application/octet-stream';
+      let mimeType = type;
```

Comment:
This can be simplified to:

```js
const MIME_TYPE_RAW_IMAGE = 'application/octet-stream';
const mimeType = _.get(handlers, type, MIME_TYPE_RAW_IMAGE);
return _.invoke(handlers, mimeType, file, {
    size: fileStats.size
});
```